create-onchain/src/cli.ts

```diff
@@ -14,7 +14,7 @@ import {
 } from './utils.js';
 
 const sourceDir = path.resolve(
-  fileURLToPath(import.meta.url), 
+  path.dirname(fileURLToPath(import.meta.url)), 
   '../../../templates/next'
```

Comment:
What OS are you using? Is this working for you locally as you have in your commit?

```suggestion
  '../../templates/next'
```

Reply:
How are you running the script?

You'll need to ensure that it works after running these commands from create-onchain root:

`bun run build`
`node dist/esm/cli.js`